module/os/freebsd/zfs/arc_os.c

```diff
@@ -226,15 +226,15 @@ arc_lowmem(void *arg __unused, int howto __unused)
 	mutex_enter(&arc_adjust_lock);
 	arc_adjust_needed = B_TRUE;
 	zthr_wakeup(arc_adjust_zthr);
-
 	/*
 	 * It is unsafe to block here in arbitrary threads, because we can come
 	 * here from ARC itself and may hold ARC locks and thus risk a deadlock
 	 * with ARC reclaim thread.
 	 */
 	if (curproc == pageproc)
-		(void) cv_wait(&arc_adjust_waiters_cv, &arc_adjust_lock);
-	mutex_exit(&arc_adjust_lock);
```

Comment:
You've removed mutex_exit(), while keeping mutex_enter() above. It seems unfinished.

Reply:
Whoops, yes I meant to remove the mutex_enter() and associated code above.  Fixed now.